Unable to build xterm-addon-image : Can't resolve 'common/Lifecycle'

## Details
- OS version: Debian testing
- xterm.js version: 5.3.0

### Steps to reproduce

1. yarn install
2. npm run install-addons
3. npm run build
4. cd addons/xterm-addon-image
5. npm run package

This produces an error:
```
ERROR in ./out/ImageRenderer.js 9:20-47
Module not found: Error: Can't resolve 'common/Lifecycle' in '/home/yadd/dev/debian/src/pkg-js/packages/node-xterm/addons/xterm-addon-image/out'
resolve 'common/Lifecycle' in '/home/yadd/dev/debian/src/pkg-js/packages/node-xterm/addons/xterm-addon-image/out'
```

node_modules/common is installed with the good version (0.2.5) and doesn't export any `common/Lifecycle`

--- a/addons/addon-image/webpack.config.js
+++ b/addons/addon-image/webpack.config.js
@@ -21,6 +21,14 @@ const addon = {
       }
     ]
   },
+  resolve: {
+    modules: ['./node_modules'],
+    extensions: [ '.js' ],
+    alias: {
+      common: path.resolve('../../out/common'),
+      browser: path.resolve('../../out/browser')
+    }
+  },
   output: {
     filename: mainFile,
     path: path.resolve('./lib'),

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -128,6 +128,9 @@ function getPublishedVersions(packageJson, version, tag) {
     throw new Error('Could not get published versions\n' + err);
   }
   const versionsJson = asArray(JSON.parse(versionsProcess.stdout));
+  if (!versionsJson) {
+    return [];
+  }
   if (tag) {
     return versionsJson.filter(v => !v.search(new RegExp(`${version}-${tag}.[0-9]+`)));
   }

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -127,10 +127,11 @@ function getPublishedVersions(packageJson, version, tag) {
     }
     throw new Error('Could not get published versions\n' + err);
   }
-  const versionsJson = asArray(JSON.parse(versionsProcess.stdout));
-  if (!versionsJson) {
+  const output = JSON.parse(versionsProcess.stdout);
+  if (!output || Array.isArray(output) && output.length === 0) {
     return [];
   }
+  const versionsJson = asArray(output);
   if (tag) {
     return versionsJson.filter(v => !v.search(new RegExp(`${version}-${tag}.[0-9]+`)));
   }

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -128,6 +128,12 @@ function getPublishedVersions(packageJson, version, tag) {
     throw new Error('Could not get published versions\n' + err);
   }
   const output = JSON.parse(versionsProcess.stdout);
+  if (typeof output === 'object') {
+    if (output.error?.code === 'E404')  {
+      return [];
+    }
+    throw new Error('Could not get published versions\n' + output);
+  }
   if (!output || Array.isArray(output) && output.length === 0) {
     return [];
   }

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -9,7 +9,7 @@ const os = require('os');
 const path = require('path');
 
 // Setup auth
-// fs.writeFileSync(`${process.env['HOME']}/.npmrc`, `//registry.npmjs.org/:_authToken=${process.env['NPM_AUTH_TOKEN']}`);
+fs.writeFileSync(`${process.env['HOME']}/.npmrc`, `//registry.npmjs.org/:_authToken=${process.env['NPM_AUTH_TOKEN']}`);
 
 const isDryRun = process.argv.includes('--dry');
 if (isDryRun) {

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -128,8 +128,6 @@ function getPublishedVersions(packageJson, version, tag) {
     throw new Error('Could not get published versions\n' + err);
   }
   const output = JSON.parse(versionsProcess.stdout);
-  console.log('output', output);
-  console.log('typeof output', typeof output);
   if (typeof output === 'object' && !Array.isArray(output)) {
     if (output.error?.code === 'E404')  {
       return [];
@@ -139,9 +137,7 @@ function getPublishedVersions(packageJson, version, tag) {
   if (!output || Array.isArray(output) && output.length === 0) {
     return [];
   }
-  console.log('output', output);
   const versionsJson = asArray(output);
-  console.log('versionsJson', versionsJson);
   if (tag) {
     return versionsJson.filter(v => !v.search(new RegExp(`${version}-${tag}.[0-9]+`)));
   }

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -9,7 +9,7 @@ const os = require('os');
 const path = require('path');
 
 // Setup auth
-fs.writeFileSync(`${process.env['HOME']}/.npmrc`, `//registry.npmjs.org/:_authToken=${process.env['NPM_AUTH_TOKEN']}`);
+// fs.writeFileSync(`${process.env['HOME']}/.npmrc`, `//registry.npmjs.org/:_authToken=${process.env['NPM_AUTH_TOKEN']}`);
 
 const isDryRun = process.argv.includes('--dry');
 if (isDryRun) {
@@ -128,7 +128,9 @@ function getPublishedVersions(packageJson, version, tag) {
     throw new Error('Could not get published versions\n' + err);
   }
   const output = JSON.parse(versionsProcess.stdout);
-  if (typeof output === 'object') {
+  console.log('output', output);
+  console.log('typeof output', typeof output);
+  if (typeof output === 'object' && !Array.isArray(output)) {
     if (output.error?.code === 'E404')  {
       return [];
     }

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -30,7 +30,7 @@ const addonPackageDirs = [
   path.resolve(__dirname, '../addons/addon-attach'),
   path.resolve(__dirname, '../addons/addon-canvas'),
   path.resolve(__dirname, '../addons/addon-fit'),
-  // path.resolve(__dirname, '../addons/addon-image'),
+  path.resolve(__dirname, '../addons/addon-image'),
   path.resolve(__dirname, '../addons/addon-ligatures'),
   path.resolve(__dirname, '../addons/addon-search'),
   path.resolve(__dirname, '../addons/addon-serialize'),

--- a/bin/publish.js
+++ b/bin/publish.js
@@ -131,7 +131,9 @@ function getPublishedVersions(packageJson, version, tag) {
   if (!output || Array.isArray(output) && output.length === 0) {
     return [];
   }
+  console.log('output', output);
   const versionsJson = asArray(output);
+  console.log('versionsJson', versionsJson);
   if (tag) {
     return versionsJson.filter(v => !v.search(new RegExp(`${version}-${tag}.[0-9]+`)));
   }

